components/announcement_bar/show_start_trial_modal/show_start_trial_modal.tsx

```diff
@@ -59,7 +59,7 @@ const ShowStartTrialModal = () => {
         if (!stats?.TOTAL_USERS) {
             dispatch(getStandardAnalytics());
         }
-    }, [!stats?.TOTAL_USERS]);
+    }, [stats?.TOTAL_USERS]);
```

Comment:
This is unrelated, right?

Reply:
yes, and a merge problem, the idea was to completely remove that dependency which is unnecessary